ml-agents/mlagents/trainers/bc/trainer.py

```diff
@@ -28,22 +28,20 @@ def __init__(self, brain, trainer_parameters, training, load, seed, run_id):
         :param seed: The seed the model will be initialized with
         :param run_id: The The identifier of the current run
         """
-        super(BCTrainer, self).__init__(brain, trainer_parameters, training, run_id)
+        super(BCTrainer, self).__init__(brain, trainer_parameters, training,
+                                        run_id)
         self.policy = BCPolicy(seed, brain, trainer_parameters, load)
         self.n_sequences = 1
         self.cumulative_rewards = {}
         self.episode_steps = {}
         self.stats = {'Losses/Cloning Loss': [], 'Environment/Episode Length': [],
                       'Environment/Cumulative Reward': []}
 
-        self.summary_path = trainer_parameters['summary_path']
         self.batches_per_epoch = trainer_parameters['batches_per_epoch']
-        if not os.path.exists(self.summary_path):
-            os.makedirs(self.summary_path)
+
 
         self.demonstration_buffer = Buffer()
         self.evaluation_buffer = Buffer()
-        self.summary_writer = tf.summary.FileWriter(self.summary_path)
```

Comment:
Did you test that BC is still training and logging tensorboard ?

Reply:
Just tried it, looks like it is still running and logging